katas/content/nonlocal_games/examples/CHSHGameDemo.qs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//////////////////////////////////////////////////////////////////////
+// This file contains reference solutions to all above exercises.
+// We recommend that you try to solve the exercises yourself first,
+// but feel free to look up the solution if you get stuck.
+//////////////////////////////////////////////////////////////////////
+
```

Comment:
We don't include the license headers in the katas demos, and we shouldn't keep the comment from the classic katas, it doesn't really make sense in the modern katas context
```suggestion
```

Reply:
All comments and license headers are removed from demo. Demo is refactored.